corehq/ex-submodules/phonelog/tasks.py

```diff
@@ -23,6 +23,6 @@ def purge_old_device_report_entries():
     UserEntry.objects.filter(server_date__lt=max_age).delete()
 
 
-@task
+@task(queue='sumologic_logs_queue')
```

Comment:
should this use a setting that defaults to the 'default' queue? I know its only on ICDS right now but I think that's how we'd want it in the future.

Reply:
Would it be better to do that or [this](https://github.com/dimagi/commcare-cloud/blob/c5a8484d944bef4adaec7043add8f3509b78b178/environments/staging/app-processes.yml#L12) ?

It might be more clear if we always have the queue the same, make it required in the celery processes, and have one worker that listens to multiple queues

Reply:
I agree with @emord's suggestion. In a recent refactor, I tried to set the convention that all queue names are hardcoded, and if we want them to be run on the same process, we state that explicitly in the config. Creates only one place to look in every env to see how the distinct queues are getting run. Drawback is that each env has to be explicit about it/updated to incorporate new queues as they come, but that drawback is mitigated by the static checks that make you declare each queue as required or optional and (I estimate) go a long way in preventing errors related to this.

Reply:
+1, I temporarily forgot about that